src/sentry/buffer/base.py

```diff
@@ -150,7 +150,7 @@ def process(
                     # continue
                     pass
                 else:
-                    group.update(**update_kwargs)
+                    group.update(using=None, **update_kwargs)
```

Comment:
this one is a bit annoying, because `using` *could* be passed as a named or positional it *could* collide with arbitrary names in `update_kwargs` so the type checker rejects it.   explicitly passing the default here appeases the checker though

Reply:
what's the annoyance? seems fine to me as it's just overridden if present in update_kwargs

Reply:
in practice we don't have fields named `using` so it couldn't possibly collide